components/Snackbar/snackbar.js

```diff
@@ -0,0 +1,21 @@
+const snackbarBaseline = document.querySelector(".btn-action-baseline");
+const snackbarLeading = document.querySelector(".btn-action-leading");
+
+snackbarBaseline.addEventListener("click", () => {
+  document.querySelector(".snackbar-baseline").classList.remove("hide");
+  document.querySelector(".snackbar-leading").classList.add("hide");
+  closeSnackbar();
+});
+
+snackbarLeading.addEventListener("click", () => {
+  document.querySelector(".snackbar-baseline").classList.add("hide");
+  document.querySelector(".snackbar-leading").classList.remove("hide");
```

Comment:
You can create a variable here for snackbar-baseline and snackbar-leading, as we are using it multiple time it would make your work much easier.

Reply:
Done